darts/models/forecasting/chronos2_model.py

```diff
@@ -0,0 +1,869 @@
+        quantile_preds = quantile_preds[:, :, :, self.user_quantile_indices]
+
+        return quantile_preds
+
+
+class Chronos2Model(FoundationModel, HuggingFaceModelMixin):
+    _repo_id = "amazon/chronos-2"
+    _repo_commit = "18128c7b4f3fd286f06d6d4efe1d252f1d2a9a7c"
+
+    # Fine-tuning is turned off for now pending proper fine-tuning support
+    # and configuration.
+    _allows_finetuning = False
+
+    def __init__(
+        self,
+        input_chunk_length: int,
+        output_chunk_length: int,
+        output_chunk_shift: int = 0,
+        enable_finetuning: bool = False,
+        likelihood: Optional[QuantileRegression] = None,
+        local_dir: Optional[Union[str, os.PathLike]] = None,
+        **kwargs,
+    ):
+        """Chronos-2 Model for Zero-Shot and Fine-Tuned Time Series Forecasting.
+
+        This is an implementation of Amazon's Chronos-2 model [1]_, [2]_, ported from
+        `amazon-science/chronos-forecasting <https://github.com/amazon-science/chronos-forecasting>`_
+        and adapted for Darts APIs. From the original authors:
+
+        "Chronos-2 is a 120M-parameter, encoder-only time series foundation model for zero-shot forecasting. It supports
+        univariate, multivariate, and covariate-informed tasks within a single architecture. Inspired by the T5 encoder,
+        Chronos-2 produces multi-step-ahead quantile forecasts and uses a group attention mechanism for efficient
+        in-context learning across related series and covariates. Trained on a combination of real-world and large-scale
+        synthetic datasets, it achieves state-of-the-art zero-shot accuracy among public models on fev-bench, GIFT-Eval,
+        and Chronos Benchmark II. Chronos-2 is also highly efficient, delivering over 300 time series forecasts per
+        second on a single A10G GPU and supporting both GPU and CPU inference."
+
+        This model supports past covariates (known for `input_chunk_length` points before prediction time),
+        and future covariates (known for `output_chunk_length` points after prediction time).
+
+        Using this model would automatically download the pre-trained model from HuggingFace Hub (amazon/chronos-2).
+        Alternatively, you can specify a local directory containing the model config and weights using the
+        ``local_dir`` parameter.
+
+        By default, this model is deterministic and outputs only the median (0.5 quantile). To enable probabilistic
+        forecasts, pass a :class:`darts.utils.likelihood_models.torch.QuantileRegression` instance to the ``likelihood``
+        parameter. The quantiles used must be a subset of those used during Chronos-2 pre-training, see below for
+        details. It is recommended to call :func`predict()` with ``num_samples >> 1`` to get meaningful results.
+
+        Fine-tuning of Chronos-2 is not supported at the moment. Setting ``enable_finetuning=True`` will raise an error.
```

Comment:
Let's simply not add the parameter until we actually support it

```suggestion
        Fine-tuning of Chronos-2 is not supported at the moment.
```